Packs/Jq/Scripts/Jq/Jq.yml

```
@@ -0,0 +1,40 @@
+args:
+- description: JSON String
+  name: value
+  required: true
+- description: JQ compatible query
+  name: query
+  required: true
+comment: "Run JQ Query. \n\nCheck these links:\n- https://stedolan.github.io/jq/manual/#Invokingjq\n\
+  - https://jqplay.org/"
+commonfields:
+  id: d69ea43f-ce77-4a54-89a9-66dd8bbc0a5b
```

Please change this to the script's name.
```suggestion
  id: jq
```